### PR TITLE
Updated CFE-266 doc work

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/CFE-423

Link to docs preview: 
1. Updated to the new v1beta1 version of ExternalDNS API:
aws - https://deploy-preview-45521--osdocs.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-aws.html
azure - https://deploy-preview-45521--osdocs.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-azure.html
gcp - https://deploy-preview-45521--osdocs.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-control-dns-records-public-managed-zone-gcp.html

2. Removed the manual step for the creation of `external-dns` namespace and extra role:
https://deploy-preview-45521--osdocs.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-installing-external-dns-operator.html

3. Migrated OLM channel update to stable-v1.0 channel:
https://deploy-preview-45521--osdocs.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-installing-external-dns-operator.html

Additional information: 
@alebedev87 PTAL and let me know your feedback.

### CODE REVIEW

**Comment by DhritiShikhar** [comment]
The PR looks good.

File: networking/external_dns_operator/nw-installing-external-dns-operator.adoc [hunk 1]
```diff
@@ -11,7 +11,6 @@ You can install the External DNS Operator using the Red Hat OpenShift Container
 .Prerequisites
```
Comment (alebedev87) [inline networking/external_dns_operator/nw-installing-external-dns-operator.adoc:11]: Whole Prerequsites chapter is not needed anymore.

**Pull request merged**